rfcs/text/0000-rfc-process.md

```diff
@@ -0,0 +1,172 @@
+
+To address these needs, the DCP RFC process defines a transparent and standard method for community members to propose and build consensus around substantial enhancements to DCP software or governance.
+
+### When is an RFC required?
+
+Any proposal that would benefit from additional review or design before being implemented is a good candidate for an RFC.
+
+- An RFC is required if the implemented design would be described in written communication to the community.
+- An RFC is required if a design impacts multiple DCP projects.
+- An RFC is required if a technical initiative (refactoring, major architectural change) impacts the community.
+- A RFC is required for all changes to the DCP formal governance, including but not limited to oversight, decision-making, conflict resolution, and community processes such as charters and RFCs. For more background on governance, see the **Model C: Delegated Governance** section in [Organization & Structure of Open Source Software Development Initiatives](https://dash.harvard.edu/bitstream/handle/1/30805146/2017-03-24_governance.pdf).  
+
+### Revising RFCs
+
+Where necessary, existing RFCs can be revised using the same process (a proposed change submitted as a RFC pull request and subject to community review and approval). **Major changes should result in a new RFC.**
+
+### What are the roles in the RFC process?
+
+- **Authors** are DCP community members.
+
+- **Shepherds** are members of *DCP PM* assigned to assist the **Author(s)** and ensure that the RFC progresses to closure.
+
+- **Approvers** for governance RFCs are the Project Leads and Product Owners from *DCP PM*.
+
+- **Approvers** for software RFCs are the Technical Leads from *DCP Architecture*. 
+
+- **Reviewers** are DCP community members, including both software developers and users.
+
+- **Science Reviewer** is the *DCP Science PM* in coordination with *HCA Science Governance*.
+
+## Detailed Design
+
+### Proposing RFCs
+
+#### **Authors**:
+
+*Before proposing, always engage the DCP community to build consensus and assess whether your proposal sparks interest and addresses a need. You may even discover potential collaborators.*
+
+  - Fork the HumanCellAtlas dcp-community repository
+  - Copy `rfcs/rfc-template.md` to `rfcs/text/0000-my-feature.md`
+  *- where "my-feature" is descriptive. Don't assign an RFC number yet.*
+  
+  - Fill in the RFC with attention to detail
+
+  - Submit a pull request.
+  
+      For software RFCs:
+      - Add the "rfc-proposed" and the appropriate software project name _(such as "Data Store")_ labels. When the RFC impacts multiple DCP software projects, then the "Architecture" label **MUST** be the project name. *If uncertain about the appropriate project name, then **_Ask a PM_** on the HCA **#dcp-project-mgmt** slack channel*
+
+      For governance RFCs:
+        
+      - Add the "rfc-proposed" and the "Governance" labels. 
```

Comment:
again, consider at least a third RFC of type "informational" to move away from buried google docs to something that can act as a more formation communication mechanism...

Reply:
We have _Informational_ [RFCs](https://tools.ietf.org/html/rfc2026#section-4.2.2) at the IETF. I also reviewed the **[Python Enhancement Process**](https://www.python.org/dev/peps/pep-0001/#pep-types) (PEP) which is the origin of the Rust process and supports this type:

`There are three kinds of PEP:`

`A Standards Track PEP describes a new feature or implementation for Python. It may also describe an interoperability standard that will be supported outside the standard library for current Python versions before a subsequent PEP adds standard library support in a future version.`

`An Informational PEP describes a Python design issue, or provides general guidelines or information to the Python community, but does not propose a new feature. Informational PEPs do not necessarily represent a Python community consensus or recommendation, so users and implementers are free to ignore Informational PEPs or follow their advice.`

`A Process PEP describes a process surrounding Python, or proposes a change to (or an event in) a process. Process PEPs are like Standards Track PEPs but apply to areas other than the Python language itself. They may propose an implementation, but not to Python's codebase; they often require community consensus; unlike Informational PEPs, they are more than recommendations, and users are typically not free to ignore them. Examples include procedures, guidelines, changes to the decision-making process, and changes to the tools or environment used in Python development. Any meta-PEP is also considered a Process PEP.`

The question is how we would want to process Informational RFCs. Generally, they have a different workflow. Would it be simple editorial review? Would approval be required? What are your thoughts? Do you have a specific case in mind?